convert2rhel/actions/system_checks/package_updates.py

```diff
@@ -79,48 +77,23 @@ def run(self):
             return
 
         if len(packages_to_update) > 0:
-            package_not_up_to_date_skip = os.environ.get("CONVERT2RHEL_OUTDATED_PACKAGE_CHECK_SKIP", None)
             package_not_up_to_date_error_message = (
                 "The system has %s package(s) not updated based on repositories defined in the system repositories.\n"
                 "List of packages to update: %s.\n\n"
                 "Not updating the packages may cause the conversion to fail.\n"
                 "Consider updating the packages before proceeding with the conversion."
                 % (len(packages_to_update), " ".join(packages_to_update))
             )
-            if not package_not_up_to_date_skip:
-                logger.warning(package_not_up_to_date_error_message)
-                self.set_result(
-                    level="OVERRIDABLE",
-                    id="OUT_OF_DATE_PACKAGES",
-                    title="Outdated packages detected",
-                    description="Please refer to the diagnosis for further information",
-                    diagnosis=package_not_up_to_date_error_message,
-                    remediations="If you wish to ignore this message, set the environment variable "
-                    "'CONVERT2RHEL_OUTDATED_PACKAGE_CHECK_SKIP' to 1.",
-                )
-                return
-
-            skip_package_not_up_to_date_message = (
-                "Detected 'CONVERT2RHEL_OUTDATED_PACKAGE_CHECK_SKIP' environment variable, we will skip "
-                "the package up-to-date check.\n"
-                "Beware, this could leave your system in a broken state."
-            )
-            logger.warning(skip_package_not_up_to_date_message)
-            self.add_message(
-                level="WARNING",
-                id="SKIP_OUTDATED_PACKAGE_CHECK",
-                title="Skip package not up to date check",
-                description=skip_package_not_up_to_date_message,
-            )
-
             logger.warning(package_not_up_to_date_error_message)
             self.add_message(
                 level="WARNING",
-                id="OUTDATED_PACKAGE_MESSAGE",
+                id="OUT_OF_DATE_PACKAGES",
                 title="Outdated packages detected",
                 description="Please refer to the diagnosis for further information",
                 diagnosis=package_not_up_to_date_error_message,
                 remediations="Run yum update to update all the packages on the system.",
             )
+            return
```

Comment:
I would say the `return` or the following `else` is not needed.

Reply:
I would vote for removing the else and keeping the return

Reply:
Alright I'll remove the else